pkg/network/ebpf/c/runtime/tracer.c

```diff
@@ -30,6 +31,36 @@
 # error "kernel version not included?"
 #endif
 
+static __always_inline __be32 rt_nexthop_bpf(struct rtable *rt) {
+    if (!rt) return 0;
+    __be32 hop = 0;
+#if LINUX_VERSION_CODE < KERNEL_VERSION(5, 2, 0)
+    bpf_probe_read(&hop, sizeof(hop), &rt->rt_gateway);
+#else
+    u8 family;
+    bpf_probe_read(&family, sizeof(family), &rt->rt_gw_family);
+    if (family == AF_INET) {
+        bpf_probe_read(&hop, sizeof(hop), &rt->rt_gw4);
+    }
+#endif
+    return hop;
+}
+
+static __always_inline __u32 get_netns_from_sock(struct sock* skp) {
```

Comment:
This helper function is no longer used, correct?

Reply:
Yes, removed.